bench-vortex/src/query_bench.rs

```diff
@@ -25,17 +25,13 @@ pub struct BenchmarkConfig {
 }
 
 /// Initialize logging/tracing for a benchmark
-pub fn setup_logging_and_tracing(
-    verbose: bool,
-    trace_file: &str,
-) -> anyhow::Result<Option<tracing_chrome::FlushGuard>> {
+pub fn setup_logging_and_tracing(verbose: bool, trace_file: &str) -> anyhow::Result<()> {
```

Comment:
this never errors (it also didn't error before). You could return nothing

Reply:
Genuinely the version I was looking at didn't create a file but now it does, ignore me